client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -131,12 +131,12 @@ export default React.createClass( {
 		const bulkManagement = ! this.state.bulkManagement;
 
 		if ( bulkManagement ) {
+			this.setBulkSelectionState( this.props.plugins, bulkManagement );
```

Comment:
We need to pass a boolean to `setBulkSelectionState` to be used as the selection state. In this case we want that value to be `true`. I choose to use `bulkManagement` here to avoid the magic boolean but I now see that the semantics are different. 

`bulkManagement` suggests whether or not we should prepare to display the bulk management tools. But it doesn't fully imply that the selection state _should_ be true. It only suggests that it _is_ true.  

I'm going to update this to use `true` to clear that up.